cmd/dockerd/daemon.go

```diff
@@ -103,7 +103,7 @@ func (cli *DaemonCli) start(opts *daemonOptions) (err error) {
 	system.InitLCOW(cli.Config.Experimental)
 
 	if err := setDefaultUmask(); err != nil {
-		return fmt.Errorf("Failed to set umask: %v", err)
```

Comment:
`setDefaultUmask()` already mentioned "failed to set umask", so this was a bit double; https://github.com/moby/moby/blob/7d4fa69e33e64cf637d6fb3a0130e8aa3cc83072/cmd/dockerd/daemon_unix.go#L29